integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-with-existing-selectorless-manifest/verify.groovy

```diff
@@ -0,0 +1,24 @@
+import io.dekorate.utils.Serialization
+import io.fabric8.kubernetes.api.model.KubernetesList
+import io.fabric8.kubernetes.api.model.Container
+import io.fabric8.kubernetes.api.model.Service
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+
+//Check that file exits
+String base = basedir
+File kubernetesYml = new File(base, "target/kubernetes/kubernetes.yml")
+assert kubernetesYml.exists()
+kubernetesYml.withInputStream { stream -> 
```

Comment:
Is there a good reason to use the `maven-invoker-way` of testing?

I really dislike them as they are impossible to debug :)

Reply:
Using existing resources (via `src/main/kubernetes` is not currently possible with QuarkusProdModeTest.

AFAIR, I tried to implement it in the past, but this would require creating a normal project structure under the hood (with a real pom, src folders etc).

Reply:
OK, understood

Reply:
> I really dislike them as they are impossible to debug :)

It's a PITA to debug the verify script. 
The groovy simplicity makes up for it (at least it does for me).

The actual build can be debugged using:

`mvn clean install  -pl :quarkus-integration-test-kubernetes-invoker -Dinvoker.test=kubernetes-with-existing-selectorless-manifest -Dinvoker.mavenExecutable=mvnDebug`

Reply:
Good to know, thanks